velox/core/PlanNode.h

```diff
@@ -522,13 +522,54 @@ class AggregationNode : public PlanNode {
     static Aggregate deserialize(const folly::dynamic& obj, void* context);
   };
 
+  // These fields are set if the aggregation has a GroupId input.
+  struct GroupingSetDescriptor {
```

Comment:
Do we need this struct? If so, let's make it private.

Reply:
These properties are for a special case of AggregationNode over GroupIdNode so this seemed reasonable. I have made them private now.